src/mqueryfront/src/QueryField.js

```diff
@@ -97,9 +16,10 @@ class QueryField extends Component {
                     <button
                         type="button"
                         className="btn btn-success btn-sm"
-                        onClick={(event) =>
-                            this.handleQuery(event, "query", "medium")
-                        }
+                        onClick={(event) => {
+                            event.preventDefault();
```

Comment:
Why are there `preventDefault`s everywhere? I think this button is not in any form, so it shouldn't change its behaviour?

Reply:
Agree, this is unnecessary for buttons.